src/frontends/pytorch/src/utils_quantize.cpp

```diff
@@ -80,9 +80,9 @@ ov::Output<ov::Node> quantize(const NodeContext& context,
         } else if (dtype == element::i8) {
             out_low_i64 = (int64_t)std::numeric_limits<char>::lowest();
             out_high_i64 = (int64_t)std::numeric_limits<char>::max();
-        } else {  // i32
-            out_low_i64 = (int64_t)std::numeric_limits<int>::lowest();
-            out_high_i64 = (int64_t)std::numeric_limits<int>::max();
+        } else { // in other cases, also force u8
```

Comment:
I am not sure whether to leave this as a separate case for reading clarity or merge it with u8 case

Reply:
This change fixes all quantized ops' inaccuracies for big values (still some rare exceptions occur though). Based on [this](http://www.xavierdupre.fr/app/mlprodict/helpsphinx/onnxops/onnx__QuantizeLinear.html) doc, 

> T2 in ( tensor(int8), tensor(uint8) ): Constrain ‘y_zero_point’ and ‘y’ to 8-bit integer tensor.

Reply:
Does it mean that not only int8 and uint8 can appear for mul?

Reply:
It takes the dtype of zero_point, if it's not quint8/qint8, it is set to quint8. For the inputs, only quint8/qint8 is allowed (for ov), since FakeQuantize doesn't work for other types anyways